src/pilot/cpu_siblings.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python
+
+# Copyright (c) 2016-2018 Dell Inc. or its subsidiaries.
```

Comment:
Can you change the copyright date to 2018?

Reply:
Fixed